src/frontend/test/features/featurefiles/e2e/UTOPIA-1121_Drafter_ViewPIA.feature

```diff
@@ -0,0 +1,73 @@
+Feature: UTOPIA-1121 Drafter:View PIA details
+
+  Scenario Outline: As <role> user, I am able to submit a valid form with all fields filled
+
+    Given I am on the login page
+    And I click on "Log in with IDIR" button
+    When I login with user <username> having role <role>
+    And I am able to click link "Create new"
+    And I am able to click link "Start PIA Intake"
+    And Check for createPIA "first" form
+    And I set scenario name
+    And I enter "Title"
+    And I enter "Branch"
+    And I enter "Lead Name"
+    And I enter "Lead Title"
+    And I enter "Lead Email"
+    And I enter "Ministry"
+    And Await SavedAt function
+    And I enter "Initiative Details"
+    And I enter "Scope Text"
+    And I enter "Data Text"
+    And I enter "Personal information as No"
+    And Await SavedAt function 
+    And I click on "List of PIAs" tab
+    And Search with title
+    And Verify if the status is only "Incomplete"
+  
+    
+
+
+    Examples:
+      | username | role    |
+      | DOND     | Drafter |
+    
+
+ Scenario Outline: As <role> user, I am able to submit a valid form with all fields filled
+
+    Given I am on the login page
+    And I click on "Log in with IDIR" button
+    When I login with user <username> having role <role>
+    And I am able to click link "Create new"
+    And I am able to click link "Start PIA Intake"
+    And Check for createPIA "first" form
+    And I set scenario name
+    And I enter "Title"
+    And I enter "Branch"
+    And I enter "Lead Name"
+    And I enter "Lead Title"
+    And I enter "Lead Email"
+    And I enter "Ministry"
+    And Await SavedAt function
+    And I enter "Initiative Details"
+    And I enter "Scope Text"
+    And I enter "Data Text"
+    And I enter "Personal information as Yes"
+    And Await SavedAt function 
+    Then I click on "Submit" button
+    And I see "Are you sure you want to submit your PIA intake?"
+    Then I click on "Yes, submit" button
+    Then I click on "Save" button
+    Then I click on "Submit" button
+    Then I click on "Yes, submit" button
+    And I click on "List of PIAs" tab
+    And Search with title
+    And Verify if the status is only "MPO Review"
```

Comment:
This is Incorrect.

If you entered "Personal information as Yes", which makes it a Standard PIA, then the app either stays in Incomplete or can move to Edit in Progress based on the Usr choice on Next Steps page. It should not move to  MPO Review. If that is happening, that would be a bug.

I can explain you the flows in the app.

Reply:
Hello @kushal-arora-fw - As discussed, this scenario is to only check the different status changes with submit button.